src/components/CommonForm/commonForm.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import { Grid, Button, Box, makeStyles, createStyles, Theme } from "@material-ui/core";
+import { IInputFields } from "../../models/index";
+import InputFields from "../InputFields/inputField";
+import { FORM_ACTIONS } from "../Forms/constant";
+
+const useStyles = makeStyles((theme: Theme) =>
+	createStyles({
+		button: {
+			color: "white",
+			marginRight: theme.spacing(2),
+		},
+	})
+);
+
+function CommonInputForm({
+	initialValues,
+	validate,
+	onCreate,
+	onUpdate,
+	onCancel,
+	inputFields,
+	formAction,
+}: any) {
+	const classes = useStyles();
+	const validateInitialValue = (initialValue: any) => {
+		const errors = validate(initialValue) as object;
+		if (!errors) return true;
+		return Object.keys(errors).length ? false : true;
+	};
+	return (
+		<Formik
+			initialValues={initialValues}
+			onSubmit={(values: any) => {
```

Comment:
Instead of using any, define a type

Reply:
Every time value will be different depending on form input fields